README.md

```diff
@@ -1,2 +1,29 @@
-# hello-project
-Set up a "Hello Microverse" project
+# Hello Microverse
+
+> The aim of this project is using linters with each project.
+
+## Built With
+
+- HTML, CSS
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/Yazino12)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
```

Comment:
- [ ] [OPTIONAL] Good job using the Microverse README template, however, you should ensure that you have no broken links as this may lead people viewing your repository such as hiring managers to get a poor impression of you. Please ensure you add the correct link to your issues page